org.scala-refactoring.library/src/main/scala/scala/tools/refactoring/common/tracing.scala

```diff
@@ -21,7 +26,32 @@ trait Tracing {
   def trace(msg: => String)
 }
 
-trait ConsoleTracing extends Tracing {
+object DebugTracing {
+  private val debugStream = {
+    Option(System.getProperty("scala.refactoring.traceFile")).flatMap { fn =>
```

Comment:
I would prefer to have the possibility to log against a stream, not against a file. Scala IDE already has a logger (see `HasLogger`), what we really want is to be able to debug everything into that logger and its associated file.